src/_P021_Level.ino

```diff
@@ -139,7 +139,7 @@ boolean Plugin_021(uint8_t function, struct EventStruct *event, String& string)
 
               if ((P021_AUTOSAVE_TIMER > 0) &&                               // - Autosave is set
                   (P021_DONT_ALWAYS_SAVE != 0) &&                            // - Save is off
-                  ((UserVar.getUint32(event->TaskIndex, 3) = 0u) ||          // - Timer not yet started or uninitialized
+                  ((UserVar.getUint32(event->TaskIndex, 3) == 0u) ||         // - Timer not yet started or uninitialized
```

Comment:
Good practice is to have the 'const' values on the left, so the compiler will error on such mistakes

Reply:
Yeah, and I had changed it to `<= 0` which is quite pointless for an unsigned int, so tried to revert that, but only removed the `<`... 😊